src/main/cms/cms_menu_imu.c

```diff
@@ -391,6 +391,73 @@ static CMS_Menu cmsx_menuFilterPerProfile = {
     .entries = cmsx_menuFilterPerProfileEntries,
 };
 
+static uint8_t cmsx_dstPidProfile;
```

Comment:
Double negatives like `#ifndef DISABLE_EXTENDED_CMS_MENUS` should not be used. The common format is more like `#ifdef USE_EXTENDED_CMS_MENUS`. Add this to `./src/main/target/common_fc_pre.h`
Also, I think all parts of this feature should be optionally selectable. Not only the CMS parts.